expression/builtin_time_vec.go

```diff
@@ -1681,11 +1682,45 @@ func (b *builtinTimestampDiffSig) vecEvalInt(input *chunk.Chunk, result *chunk.C
 }
 
 func (b *builtinUnixTimestampIntSig) vectorized() bool {
-	return false
+	return true
 }
 
+// evalInt evals a UNIX_TIMESTAMP(time).
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_unix-timestamp
 func (b *builtinUnixTimestampIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETDatetime, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalTime(b.ctx, input, buf); err != nil && terror.ErrorEqual(types.ErrInvalidTimeFormat.GenWithStackByArgs(buf), err) {
+		return err
```

Comment:
We'd better keep the compatible here. If an error occurs, the best way is calling the non-vectorized method `EvalInt`, for example,
```go
i64s := result.Int64s()
if err := b.args[0].VecEvalTime(b.ctx, input, buf); err != nil && terror.ErrorEqual(types.ErrInvalidTimeFormat.GenWithStackByArgs(buf), err) {
    var isNull bool
    for i := 0; i < n; i++ {
        i64s[i], isNull, err = b.evalInt(input.GetRow(i))
        if err != nil { ... }
        if isNull { result.SetNull(i) }
    }
}
```